build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.5
+FROM golang:1.5.1
```

Comment:
Ah, that must be what happened. Is there any danger to being explicit with the build number here?